### PR TITLE
Remove references to hydra.iohk.io from CONTRIBUTING.adoc

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -65,16 +65,16 @@ caches have not been configured correctly.
...
 extra-experimental-features = nix-command flakes
 ----
 
 On NixOS, set the following NixOS options:
 ----
 nix = {
-  binaryCaches          = [ "https://hydra.iohk.io" "https://cache.iog.io" ];
-  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" ];
+  binaryCaches          = [ "https://cache.iog.io" "https://cache.nixos.org"];
+  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" ];
```
Comment (LGLO) [inline CONTRIBUTING.adoc:77]: I think you should also update trusted key here: remove one of hydra but add one for cache.iog.io.

cache.iog.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= is my best known value of it

Comment (michaelpj) [reply CONTRIBUTING.adoc:77]: I wasn't aware we had a signing key for `cache.iog.io`? We probably want to keep the hydra one indefinitely: that cache probably has a bunch of stuff that was built by hydra, I don't know if it has been re-signed by `cache.iog.io`?

**Review by michaelpj — APPROVED** [review]

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (LGLO) [reply CONTRIBUTING.adoc:77]: I'm sorry. Seems my comment was invalid.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -65,16 +65,16 @@ caches have not been configured correctly.
...
 trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
 extra-experimental-features = nix-command flakes
 ----
 
 On NixOS, set the following NixOS options:
 ----
 nix = {
-  binaryCaches          = [ "https://hydra.iohk.io" "https://cache.iog.io" ];
-  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" ];
+  binaryCaches          = [ "https://cache.iog.io" "https://cache.nixos.org"];
```
Comment (michaelpj) [inline CONTRIBUTING.adoc:76]: I think you don't need to do this? the nixos options here are unioned, so the default setting of `cache.iog.io` will not be overwritten. Or am I forgetting?

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (michaelpj) [reply CONTRIBUTING.adoc:77]: Does what you wrote work?

File: CONTRIBUTING.adoc (hunk 2 above)
Comment (zeme-iohk) [reply CONTRIBUTING.adoc:76]: You mean the default settings of `https://cache.nixos.org`? 
In general they are merged yes, so we don't need that. Though it doesn't hurt to keep.
I'll remove it.

Comment (michaelpj) [reply CONTRIBUTING.adoc]: As someone who knows how nixos works it looked wrong to me, I think it's better to write the idiomatic thing!

**Review by michaelpj — APPROVED** [review]

**Pull request merged**